feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.service.spec.ts

```diff
@@ -61,6 +61,115 @@ const orderEntry: OrderEntry = {
   ],
 };
 
+const productName0: string = 'PRODUCT_NAME_0';
```

Comment:
Please remove the un-needed type qualifiers. I assume that tslint will also complain about them (Also for the following lines).
Correction: tslint does not complain, still I would recommend to remove it for conciseness. But it does not matter if we don't catch all.

```suggestion
const productName0 = 'PRODUCT_NAME_0';
```